setup.py

```diff
@@ -18,15 +24,10 @@ def readme():
     packages=find_packages(),
     python_requires=">=3.6",
     license="Apache 2.0",
-    install_requires=[
-        "click==7.0",
-        "tensorflow-datasets==1.0.2",
-        "larq==0.2.0",
-        "zookeeper==0.1.1",
-    ],
+    install_requires=install_requires(),
     extras_require={
-        "tensorflow": ["tensorflow==1.14.0rc0"],
-        "tensorflow_gpu": ["tensorflow-gpu==1.14.0rc0"],
+        "tensorflow": ["tensorflow==1.14.0"],
```

Comment:
We ran the latest experiments with TF 2, correct?

```suggestion
        "tensorflow": ["tensorflow==2.0.0"],
```